crates/uv/src/lib.rs

```diff
@@ -1511,6 +1523,7 @@ async fn run_project(
                 globals.connectivity,
                 globals.concurrency,
                 globals.native_tls,
+                &globals.allow_insecure_host,
```

Comment:
Should we pass in the owned value? We're cloning it at all the sites, aren't we?